node/src/main/scala/com/wavesplatform/api/http/utils/UtilsInvocationRequest.scala

```diff
@@ -0,0 +1,47 @@
+package com.wavesplatform.api.http.utils
+
+import cats.implicits.{toBifunctorOps, toTraverseOps}
+import com.wavesplatform.account.{AddressOrAlias, PublicKey}
+import com.wavesplatform.api.http.requests.InvokeScriptRequest
+import com.wavesplatform.api.http.requests.InvokeScriptRequest.FunctionCallPart
+import com.wavesplatform.common.state.ByteStr
+import com.wavesplatform.lang.ValidationError
+import com.wavesplatform.lang.directives.values.V6
+import com.wavesplatform.lang.v1.evaluator.ContractEvaluator.Invocation
+import com.wavesplatform.lang.v1.traits.domain.Recipient.Address
+import com.wavesplatform.transaction.TxValidationError.GenericError
+import com.wavesplatform.transaction.smart
+import com.wavesplatform.transaction.smart.InvokeScriptTransaction.Payment
+import com.wavesplatform.transaction.smart.{AttachedPaymentExtractor, InvokeTransaction}
+import play.api.libs.json.{Json, Reads}
+
+case class UtilsInvocationRequest(
+    call: Option[FunctionCallPart],
+    id: Option[String],
+    fee: Option[Long],
+    feeAssetId: Option[String],
+    sender: Option[String],
+    senderPublicKey: Option[String],
+    payment: Option[Seq[Payment]]
+) {
+  def toInvocation: Either[ValidationError, Invocation] =
```

Comment:
You can use default values for some fields, see https://scastie.scala-lang.org/Ck1UwdqATkqBfF0Kq2YQow